src/main/java/teammates/ui/controller/ActionFactory.java

```diff
@@ -120,6 +120,8 @@
         map(STUDENT_FEEDBACK_RESULTS_PAGE, StudentFeedbackResultsPageAction.class);
         map(STUDENT_FEEDBACK_SUBMISSION_EDIT_PAGE, StudentFeedbackSubmissionEditPageAction.class);
         map(STUDENT_FEEDBACK_SUBMISSION_EDIT_SAVE, StudentFeedbackSubmissionEditSaveAction.class);
+        map(STUDENT_FEEDBACK_RESPONSE_COMMENT_ADD, StudentFeedbackResponseCommentAddAction.class);
```

Comment:
On another thought `StudentFeedbackResponseCommentAddAction.class`is not needed and neither edit action because student comments are being handled in `FeedbackSubmissionEditSaveAction.java `, (Extraction logic)

Reply:
I believe that enable student to add comment without submitting the whole form.

Reply:
In that case, suppose user doesn't give response and just tries to add comment. How can this scenario be taken care of?
One of the ways that I can think of is deleting comments from datastore when whole form is submitted, and response is found to be empty.

Reply:
Ok, let's keep thing simple. So instructor side can actually remain untouched. And student can only submit comment via `FeedbackSubmissionEditSaveAction.java`?